sdk/core/System.ClientModel/tests/TestFramework/Mocks/MockPipelineResponse.cs

```diff
@@ -50,6 +53,31 @@ public MockPipelineResponse SetContent(string content)
         set => _contentStream = value;
     }
 
+    public override BinaryData Content
+    {
+        get
+        {
+            if (_contentStream is null)
+            {
+                return BinaryData.FromString("");
```

Comment:
why fromstring? Why not from empty array?

Reply:
This was a mock so I was being lazy, but I can change it.